samples/src/main/java/ecom/EcomAsyncJavaClient.java

```diff
@@ -35,10 +41,14 @@ public EcomAsyncJavaClient() {
     }
 
     public CompletableFuture<PaymentInitiationResponse> sendPaymentAsync(long amount) {
-        LOG.info("Initiating payment for amount: {}", amount);
         String paymentReference = generatePaymentReference();
         String orderId = generateOrderId();
         String currency = CurrencyEnum.HKD.name();
```

Comment:
This should match the store configuration, so maybe it should go into the `config.properties` file

Reply:
Oh, I thought it can be modified per payment... 
Is that on the roadmap? Our webshop can accept multiple currencies.

Reply:
@sishbi ?

Reply:
A store should only use a single currency...

Reply:
I think... but maybe I am wrong?
I don't know if the currency is passed down to the payment request...